tests/components/zha/test_websocket_api.py

```diff
@@ -440,18 +441,102 @@ async def test_list_groupable_devices(
+    assert added_group["members"] == [
+        {
+            "device": {
+                "active_coordinator": False,
+                "area_id": None,
+                "available": True,
+                "device_reg_id": groupable_device.device_id,
+                "device_type": "EndDevice",
+                "endpoint_names": [
+                    {
+                        "name": "ON_OFF_SWITCH",
+                    },
+                ],
+                "entities": [
+                    {
+                        "entity_id": "switch.fakemanufacturer_fakemodel_switch_2",
+                        "name": "FakeManufacturer FakeModel",
+                    },
+                ],
+                "ieee": IEEE_GROUPABLE_DEVICE,
+                "last_seen": time.strftime(
+                    "%Y-%m-%dT%H:%M:%S",
+                    time.localtime(groupable_device.device.last_seen),
+                ),
+                "lqi": None,
+                "manufacturer": "FakeManufacturer",
+                "manufacturer_code": 4151,
+                "model": "FakeModel",
+                "name": "FakeManufacturer FakeModel",
+                "neighbors": [],
+                "nwk": 47004,
+                "power_source": "Battery or Unknown",
+                "quirk_applied": False,
+                "quirk_class": "zigpy.device.Device",
+                "quirk_id": None,
+                "routes": [],
+                "rssi": None,
+                "signature": {
+                    "endpoints": {
+                        "1": {
+                            "device_type": "0x0000",
+                            "input_clusters": [
+                                "0x0000",
+                                "0x0004",
+                                "0x0006",
+                            ],
+                            "output_clusters": [],
+                            "profile_id": "0x0104",
+                        },
+                    },
+                    "manufacturer": "FakeManufacturer",
+                    "model": "FakeModel",
+                    "node_descriptor": {
+                        "aps_flags": 0,
+                        "complex_descriptor_available": 0,
+                        "descriptor_capability_field": 0,
+                        "frequency_band": 8,
+                        "logical_type": 2,
+                        "mac_capability_flags": 128,
+                        "manufacturer_code": 4151,
+                        "maximum_buffer_size": 127,
+                        "maximum_incoming_transfer_size": 100,
+                        "maximum_outgoing_transfer_size": 100,
+                        "reserved": 0,
+                        "server_mask": 10752,
+                        "user_descriptor_available": 0,
+                    },
+                },
+                "user_given_name": None,
+            },
+            "endpoint_id": 1,
+            "entities": [],
+        }
+    ]
```

Comment:
Also seen other reviews mentioning that we might not want something this long in tests and that (ambr) snapshots should be used in those cases. Of course, it's only one member here..

Do we actually need to check the whole device with signature and so on here? Wouldn't it be enough to just check the `ieee` and maybe also `device_reg_id` for example?

Reply:
seems reasonable to me. updated!